src/seedu/addressbook/commands/Command.java

```diff
@@ -63,6 +63,8 @@ protected ReadOnlyPerson getTargetPerson() throws IndexOutOfBoundsException {
         return relevantPersons.get(getTargetIndex() - DISPLAYED_INDEX_OFFSET);
     }
 
+    abstract public  boolean isMutating();
```

Comment:
Need to add header comments to this method. If not, the client who is overriding this will not know what to expect.